src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -192,9 +198,10 @@ public boolean hasResident(Resident resident) {
 		return residents.contains(resident);
 	}
 
+	@Deprecated
 	public boolean hasAssistant(Resident resident) {
```

Comment:
When you deprecate something you provide information as to what version it was deprecated as well as alternative methods in the javadoc for example:

```java
/**
* @deprecated Since VERSION, user {@link myNewMethod} instead.
*
*/
public void myDeprecatedMethod(){}
```

Reply:
What Javadoc? Most of Towny, including (`Town.hasAssistant()`) is
undocumented and the replacement is obviously the code the method is
functioning as an alias to for.

Reply:
What Javadoc? Most of Towny, including (`Town.hasAssistant()`) is
undocumented and the replacement is obviously the code the method is
functioning as an alias to for.

Reply:
Resolved by commit b1b7da5dddceb3fdae695faabceaf1d4e5edc5b9.